CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/MapViewTopComponent.java

```diff
@@ -190,8 +191,9 @@ protected void handleNewGraph(final Graph graph) {
                 CompletableFuture.runAsync(() -> {
                     try {
                         PluginExecution.withPlugin(new ExtractCoordsFromGraphPlugin(self)).executeNow(graph);
-                    } catch (final Exception e) {
-                        LOGGER.log(Level.SEVERE, e.getMessage());
+                    } catch (final PluginException | InterruptedException e) {
+                        LOGGER.log(Level.SEVERE, "Exception thrown: {0}", e.getMessage());
+                        Thread.currentThread().interrupt();
```

Comment:
separate these conditions out as the `PluginException` doesn't need the thread interrupt call